pkg/redshift

```diff
@@ -0,0 +1,23 @@
+[mirrors]
+https://github.com/jonls/redshift/archive/b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe.tar.gz
+
+[vars]
+tarball=b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe.tar.gz
+tardir=redshift-b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe
+filesize=639073
+sha512=46c49a385ed0f05a2c0f2dd359d728728c8b21f5d40cf0da8820da5c2d8711ae5a562e126a3c8604c3f15705f08973251c277dc0261ad9fc1150b169b1e7b7be
+pkgver=1
+
+[deps]
+intltool
+
+[build]
+
+echo prefix=/ > config.mak
```

Comment:
i guess we should use echo "prefix=$butch_prefix" here, so it doesnt break for users having another prefix... but it is kinda weird that later on configure is called again...

Reply:
what about this here ^